Sources/SwiftProtobuf/BinaryEncodingOptions.swift

```diff
@@ -0,0 +1,32 @@
+// Sources/SwiftProtobuf/BinaryEncodingOptions.swift - Binary encoding options
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the project authors
```

Comment:
This should probably be 2014-2023 to get the current year?

Reply:
👍🏽 copied from another file (files don't seem to be consistent), but will update